main.py

```diff
@@ -28,3 +30,15 @@ def scrape_url(url):
         return response.text
     except requests.exceptions.RequestException as err:
         raise HTTPException(status_code=400, detail=str(err))
+
+
+def extract_info_with_chatgpt(html_content):
+    openai.api_key = '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'
```

Comment:
Update the api key to this one: sk-proj-lMvFhax339FEjX8kDn48T3BlbkFJ43tDzL35aQ74okB2ZTFw